### PR TITLE
Users: Adds user details tab and save/update functionality

### PR DESCRIPTION
**Pull request by jenny-s51**
## Motivation

https://issues.redhat.com/browse/APPDUX-864

## Brief Description

Adds details tab to users section. 

## Verification Steps

1. Go to Users section.
2. Click "Add User".
3. Fill in all fields of the user form.
4. Click "Create".
5. Find the user in the data list and click on the user.
6. Verify that the information in the "Details" tab matches the information you entered.
7. Change some values in the "Details" tab and click "Save".
8. Go back to the data list. 
9. Click on the user again and verify that the changes you made have been updated in the "Details" tab.
10. Run cypress tests.

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [x] axe report has been run and resulting a11y issues have been resolved
- [x] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'
ce/output.

### CODE REVIEW

File: cypress/support/pages/admin_console/manage/users/UserDetailsPage.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    firstNameInput: string;
+    firstNameValue: string;
+    lastNameInput: string;
+    lastNameValue: string;
+    enabledSwitch: string;
+    enabledValue: boolean
+    requiredUserActions: RequiredActionAlias[];
+  
+    constructor() {
+      this.saveBtn = "[data-testid=save-user]";
```
Comment (edewit) [inline cypress/support/pages/admin_console/manage/users/UserDetailsPage.ts:17]: I've introduced an extension method for this you can set this to:
```suggestion
      this.saveBtn = "save-user";
```

File: cypress/support/pages/admin_console/manage/users/UserDetailsPage.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    cy.get(this.firstNameInput).type(this.firstNameValue);
+    cy.get(this.lastNameInput).type(this.lastNameValue);
+    cy.get(this.enabledSwitch).check({ force: true });
+
+  
+      return this;
+    }
+  
+    save() {
+      cy.get(this.saveBtn).click();
```
Comment (edewit) [inline cypress/support/pages/admin_console/manage/users/UserDetailsPage.ts:43]: and here where you use it use:
```suggestion
      cy.getId(this.saveBtn).click();
```

**Review by edewit — CHANGES_REQUESTED** [review]
Very good PR nice work

**Review by mfrances17 — APPROVED** [review]
LGTM - everything worked great, including the tests

**Review by edewit — APPROVED** [review]

**Pull request merged**